package.json

```diff
@@ -66,16 +66,16 @@
     "gulp-terser": "^1.2.0",
     "gulp-tslint": "^8.1.4",
     "gulp-typescript": "^5.0.1",
-    "jest": "^26.0.1",
+    "jest": "^26.6",
     "jest-junit": "^12.0.0",
     "ocsp": "^1.2.0",
     "request": "^2.88.0",
     "rimraf": "^3.0.2",
     "semver": "^6.3.0",
     "source-map-loader": "^0.2.4",
-    "ts-jest": "^25.5.1",
+    "ts-jest": "^26.4.4",
     "tslint": "^5.20.1",
-    "typescript": "^3.5.3",
+    "typescript": "^3.8",
```

Comment:
>"typescript": "^3.8", [](start = 4, length = 21)

Are these two minor releases .d.ts file compatable?

Reply:
How can I check whether it is or not?

Reply:
reverted